docs/upgrade-guide.md

```diff
@@ -6,6 +6,7 @@
 
 - `CartoBQTilerLayer` is removed. Use `CartoLayer` instead with `type` set to `MAP_TYPES.TILESET`.
```

Comment:
Specifically move these to the bottom of the upgrade guide

Reply:
Moved to bottom and placed in separate section